blog/content/post/2020-06-15-qubes_summation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Qubes OS & 3mdeb 'minisummit' 2020 summation
+abstract: 'The second Qubes OS & 3mdeb minisummit is ahead of us. We had gone through four evenings of topics devoted to Qubes OS, so it is time for broad summation of the event.'
+cover: /covers/qubes&3mdeb_logo.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-06-15
+archives: "2020"
+
+tags:
+  - QubesOS
+  - minisummit
+  - live
+  - YouTube
+  - firmware
+  - coreboot
+  - security
+  - TPM
+categories:
+  - Miscellaneous
+  - Security
+  - Firmware
+
+
+---
+The second Qubes OS and 3mdeb 'minisummit' is ahead of us. We have gone through
+four evenings of topics devoted to Qubes OS. It has been another fruitful and
+productive event, during which we have exchanged loads of knowledge and
+expertise. Below you will find list of topics, we have discussed with related
+videos and broad report of each day.
+
+
+* 20.05
+  1. Michał Żygowski: Qubes OS on modern AMD platform
+    * [recording](https://www.youtube.com/watch?v=Rw7rAPPyPPc&t=31s)
+  2. Norbert Kamiński: Status fwupd/LVFS support for Qubes OS
+    * [recording](https://www.youtube.com/watch?v=o_IdERo3aiE&t=984s)
+  3. AMA session
+    * [recording](https://www.youtube.com/watch?v=BSGUcW6QDYU&t=1509s)
+
+* 28.05
+  1. Piotr Król: SRTM for Qubes OS VMs
+    * [recording](https://www.youtube.com/watch?v=Eip5Rts6S2I&t=2s)
+  2. Michał Żygowski: Anti Evil Maid for modern AMD UEFI-based platform
+    * [recording](https://youtu.be/rM0vRi6qABE?t=3)
+  3. AMA session
+    * [recording](https://youtu.be/rM0vRi6qABE?t=1904)
+
+* 04.06
+  1. Piotr Król: DRTM for Qubes OS VMs
+    * [recording](https://youtu.be/pZF-jyJWTE4)
+  2. Michał Żygowski: Anti Evil Maid for Intel coreboot-based platform
+    * [recording](https://youtu.be/YE2FbFlszI4?t=9)
+  3. AMA session
+    * [recording](https://youtu.be/YE2FbFlszI4?t=1725)
+* 10.06
```

Comment:
@KBanecka do we have recordings already?

Reply:
I've added recordings.